spark/src/main/java/org/apache/iceberg/spark/actions/BaseExpireSnapshotsSparkAction.java

```diff
@@ -29,7 +29,6 @@
 import org.apache.iceberg.TableMetadata;
 import org.apache.iceberg.TableOperations;
 import org.apache.iceberg.actions.BaseExpireSnapshotsActionResult;
-import org.apache.iceberg.actions.BaseSparkAction;
```

Comment:
Unrelated to this PR? Not a big deal might as well get it in though.

Reply:
I either had to make one more class public or move the classes and clean up the imports. I should probably revert these changes and do that in a follow-up PR. Let me fix this.

Reply:
Fixed.